tests/test_api_build.py

```diff
@@ -1342,6 +1267,8 @@ def test_no_force_upload_condarc_setting(mocker, testing_workdir, testing_metada
 
 
 @pytest.mark.sanity
+@pytest.mark.skipif(sys.version[:3] <= "3.5", reason="Package does not build with Python 3.5")
+@pytest.mark.skipif(sys.version[:3] <= "3.7", reason="Not supporting versions of Python <= 3.7")
```

Comment:
We have not dropped support for Python 3.7 yet. Plus this skipif statement doesn't serve a purpose.

```suggestion
```